src/rpc/masternode.cpp

```diff
@@ -107,6 +108,8 @@ UniValue initmasternode(const JSONRPCRequest& request)
         return "success";
     }
     // legacy
+    if (request.params.size() < 2) throw JSONRPCError(RPC_MISC_ERROR, "Must specify the IP address for legacy mn");
```

Comment:
better to use `RPC_INVALID_PARAMETER` for missing parameters.

Reply:
yeah, for the error empty MN key error that is above.